app/lib/report_helper.rb

```diff
@@ -957,7 +957,8 @@ def generate_report_csv(tax_details)
   def report_csv_from_params(sample, params)
     params[:is_csv] = 1
     params[:sort_by] = "highest_nt_aggregatescore"
-    background_id = params[:background_id] || sample.default_background_id
+    background_id = params[:background_id] || params["background_id"] || sample.default_background_id
```

Comment:
Why both symbol and string?

Reply:
Tweaked. Apparently when it goes through like `Resque.enqueue(GenerateProjectReportsCsv, params)` the 'params' changes from ActionController::Parameters to 'Hash' without indifferent access for symbol vs. string. So added `hash =  HashWithIndifferentAccess.new(hash)`